draft-ietf-rats-architecture.md

```diff
@@ -1058,7 +1058,7 @@ or might be defined relative to some other timestamp or timeticks counter.
 | VG | Value generation            | A value to appear in a Claim was created.
 | AA | Attester awareness          | An Attesting Environment starts to be aware of a new/changed Claim value.
 | CC | Claim Collection            | An Attesting Environment collects a new/changed Claim value to appear in Evidence.
-| HD | Handle distribution         | A centrally generated identifier for time-bound recentness across a domain of devices is successfully distributed to Attesters.
+| HD | Handle distributed          | The (local) time when a centrally generated handle for time-bound freshness is successfully received by each entity in a domain.
```

Comment:
the term "domain" is undefined
```suggestion
| HR | Handle received          | A centrally generated handle for time-bound freshness is received.
```
None of the others start with "The time when", it's all about the event not the time.
Also if it's time received, then "distributed" is the wrong term, "received" should be used.

Reply:
The relevant part here is the distribution of handles. This allows for implicit time-keeping only: knowledge about "yes I have knowledge of that handle or not" is the only relevant part here. Therefore, the focus here is on very implicit time measures. Naturally they are basically on the other end of the spectrum that are explicit timestamps. "Awareness of changes" & "Reception of Handle" are the relevant parts here. It is also important that the nature of the Handle itself, be it Nonces or Timestamps, or other concepts is explicitly allowed and encouraged. That is the essence of the example: Triggers based on implicit time-keeping, such as awareness of changes -> the reception of handles is one of them, the changes of values another.

=> Trustable freshness with the least amount of clocks used (but you still can use clocks!)

In consequence: I approve your change. In each iteration we try to align your input with the concept itself.